R/AR/target_counts.R

```diff
@@ -62,6 +62,8 @@ get_target_counts <- function(bam_dir, reference_fasta, bed_file, min_mapq=20){
 get_coverage_files <- function(my_counts, file_dir){
   fixed_columns <- c('chromosome', 'start', 'end', 'GC', 'names')
   coverage_columns <- get_coverage_columns(fixed_columns, my_counts)
+  # add `#` as a first character to the column names - to comment the header (needed by the Bedtools intersect)
+  coverage_columns[1] <- paste("#", coverage_columns[1], sep="")
```

Comment:
This won't work and will lead to an error in line 68. I think you want to add this to the column names of the sample_df (after line 68). 

FYI, the coverage_columns variable refers to the column names from the dataframe that contain the coverages. So it contains only a subset of columns from the sample_df (which is the dataframe that is written to the file).